lms/routes.py

```diff
@@ -131,6 +131,9 @@ def includeme(config):  # pylint:disable=too-many-statements
     )
     config.add_route("vitalsource_api.launch_url", "/api/vitalsource/launch_url")
 
+    config.add_route("email.unsubscribe", "/email/unsubscribe/{token}")
```

Comment:
Should the token be in a query parameter rather than in the path? I think for example lots of logging tools will automatically strip query params for security. We might also want to pick a common authentication token query param name for this reason

Reply:
The token is not secret itself, it would be useful to have access to it in case anything goes wrong.

Reply:
Surely the token is secret? It's an authentication token: given a token for your account I can generate a URL and unsubscribe you. Given a whole lot of tokens from some leaked logs or something I can unsubscribe a whole lot of people - disaster!

And we certainly wouldn't want any tools or intermediate caches or whatever finding these URLs in some logs or something and visting them!